OWNERS_ALIASES

```diff
@@ -69,7 +69,7 @@ aliases:
   - pierDipi
   - vaikas
   knative-admin:
-  - RichieEscarez
+  - ZhiminXiang
```

Comment:
I don't actually think we should be making any changes as part of this PR (for the release) to the knative-admin. Only the knative-release-leads should change.
Revisiting the knative-admin roles (I think) should be done as a separate PR.

Reply:
hmm this is automatically generated.... maybe there is a bug in the script?

Reply:
nah, you're right as usual. Been a long day. Sorry for the noise :)